.travis.yml

```diff
@@ -15,6 +15,7 @@ notifications:
 language: php
 
 php:
+  - '7.0'
```

Comment:
Do we really need to run extra set of tests for PHP 7.0? I'm afraid it may slow things down... Since our PHP side is not so fat yet, I'd set php to the lowest supported version here, which is 7.0.

Reply:
My intuition says we should check both. I've already received different kinds of errors only on specific PHP versions (7.0, 7.1, 7.2). But that will slow down Travis ofc as you already said.

There will always be a tradeoff.

Reply:
My dream is to have nightly build run that would check everything including performance metrics, and a lightweight build that we currently have for PRs...